common/i18n/resources/ru.js

```diff
@@ -19,6 +19,10 @@ module.exports.translation = {
     unknown: 'Отсутствует/неизвестна',
   },
 
+  unit: {
+    kmSq: 'км²',
```

Comment:
this is not correct.
either add all English for now, or none

Reply:
Ok!